.travis.yml

```diff
@@ -48,6 +48,11 @@ install:
   # libimagequant
   - pushd depends && ./install_imagequant.sh && popd
 
+before_script:
+# QT needs a display for some of the tests, and it's only run on the system site packages install
```

Comment:
"Qt" rather than "QT". 

And is this needed for all build jobs, or can we limit it to just 2.7_with_system_site_packages?